variables.tf

```diff
@@ -3,9 +3,9 @@ variable "vault_addr" {
 }
 
 variable "bootstrap_version" {
-  default = "v2.2.0"
+  default = "v2.6.1"
```

Comment:
There is a [PR](https://github.com/aeternity/infrastructure/pull/407) which might bump the version. However it is not blocking this PR